_docs/schema/schemagen/schema-generation.md

```diff
@@ -70,7 +70,7 @@ All of these and more are supplied via a set of attributes that can be applied t
 
 \* The `[Obsolete]` attribute is `System.Obsolete`.  All of the others have been defined within this library.  `System.ComponentModel.DataAnnotations` support is currently [in discussion](https://github.com/gregsdennis/json-everything/issues/143).
 
-\*\* The `[JsonExclude]` attribute functions equivalently to `[JsonIgnore]` (see below).  It is included to allow generation to skip a property while allowing serialization to consider it.
+\*\* The `[JsonExclude]` attribute functions equivalently to `[JsonIgnore]` (see below).  It is included to allow generation to skip a property or an enum member while allowing serialization to consider it.
```

Comment:
Did you change `[JsonExclude]` to work on enum members?  That's what this text is about.  If not, then `[JsonExclude]` still only operates on properties.

Reply:
It works on both. I used the same method of applying both to enum members as you had on properties.